pandas/core/internals/blocks.py

```diff
@@ -575,13 +575,10 @@ def replace(
         values = self.values
 
         if isinstance(values, Categorical):
-            # TODO: avoid special-casing
```

Comment:
we still want to avoid special-casing Categorical.  This _may_ mean implementing _replace on other EAs.  Or just declaring that Categorical is special.

Reply:
Makes sense, I've added the TODO comment back.